pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -54,6 +35,8 @@ in stdenv.mkDerivation rec {
       coreutils # meson calls date, stat etc.
       glibcLocales
       patchelf getent m4
+
+      (python3Packages.python.withPackages ( ps: with ps; [ python3Packages.lxml ]))
```

Comment:
I should have a look if splicing works as expected in the cross-compiling case. However I think it only did not work before because the python env was stringified into `preConfigure`.

Reply:
Ok. Splicing did not work. I explicitly changed it back to `buildPackages`